src/components/MobileNavigation/MobileNavigation.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { ThemeContextProvider, useThemeContext } from '~/contexts';
+import {
+  useEscapeKeyListener,
+  useOverflowHidden,
+  useTrapFocus,
+} from '~/customHooks';
+import { Transition } from '~/components/Transition';
+import { MobileNavigationContextProvider } from './MobileNavigation.context';
+import {
+  ArticleList,
+  Header,
+  PrimaryNavigation,
+  SecondaryNavigation,
+} from './components';
+import type { MobileNavigationType } from './MobileNavigation.types';
+import styles from './MobileNavigation.module.css';
+
+const MobileNavigation: MobileNavigationType = ({
+  articles,
+  className,
+  closedTheme,
+  isVisuallyObstructed,
+  items,
+  header,
+  secondaryItems,
+  theme,
+}) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [activeCollectionId, setActiveCollectionId] = useState('top');
```

Comment:
Just for consistency
```suggestion
  const [isOpen, setIsOpen] = useState(false);
  const [activeCollectionId, setActiveCollectionId] = useState('top');
```